frontend/src/features/projects/common/ApprovalTransitionPage.tsx

```diff
@@ -43,15 +43,17 @@ const ApprovalTransitionPage: React.FunctionComponent<IApprovalTransitionPagePro
       return;
     }
     if (project !== undefined && !isTransitioned) {
+      // Look for a possible transition within the same workflow.
+      const next = toStatus?.filter(s => s.workflowCode === project.workflowCode);
```

Comment:
This resolves the issue when there are multiple possible transitions.  The assumption is this component will only auto transition to a status within the same workflow.